src/language/lang_ru.php

```diff
@@ -0,0 +1,659 @@
+  'Import Teams' =>
+    'Импортировать Команды',
+  'Export Teams' =>
+    'Экспортировать Команды',
+  'Import Logos' =>
+    'Импортировать Логитипы',
+  'Export Logos' =>
+    'Экспортировать Логотипы',
+  'Import Levels' =>
+    'Импортировать Уровни',
+  'Export Levels' =>
+    'Экспортировать Уровни',
+  'Import Categories' =>
+    'Импоритровать Категории',
+  'Export Categories' =>
+    'Экспортировать Категории',
+  'Levels' =>
+    'Уровни',
+  'New Quiz Level' =>
+    'Новый Уровень Викторины',
+  'Title' =>
+    'Название',
+  'Question' =>
+    'Вопрос',
+  'Level title' =>
+    'Название уровня',
+  'Quiz question' =>
+    'Вопрос викторины',
+  'Country' =>
+    'Государство',
+  'Answer' =>
+    'Ответ',
+  'Points' =>
+    'Очки',
+  'Hint' =>
+    'Подсказка',
+  'Hint Penalty' =>
+    'Штраф за Подсказку',
+  'EDIT' =>
+    'РЕДАКТИРОВАТЬ',
+  'All Quiz Levels' =>
+    'Все Уровни Викторины',
+  'Filter By:' =>
+    'Отфильтровать По:',
+  'All Status' =>
+    'Статус Всего',
+  'Enabled' =>
+    'Включено',
+  'Disabled' =>
+    'Выключено',
+  'Quiz Level' =>
+    'Уровень Викторины',
+  'Show Answer' =>
+    'Показать Ответ',
+  'Bonus' =>
+    'Бонус',
+  '-Dec' =>
+    '-Уменьшение',
+  'Save' =>
+    'Сохранить',
+  'Quiz Management' =>
+    'Управление Викториной',
+  'Add Quiz Level' =>
+    'Добавить Уровень Викторины',
+  'New Flag Level' =>
+    'Новый Уровень Флага',
+  'Description' =>
+    'Описание',
+  'Level description' =>
+    'Описание уровня',
+  'Category' =>
+    'Категория',
+  'Flag' =>
+    'Флаг',
+  'flag' =>
+    'флаг',
+  'All Flag Levels' =>
+    'Все Уровни с Флагом',
+  'New Attachment:' =>
+    'Новое Вложение:',
+  'Attachment' =>
+    'Вложение',
+  'Link' =>
+    'Ссылка',
+  'New Link:' =>
+    'Новая Сылка:',
+  'Flag Level' =>
+    'Уровень Флага',
+  'Categories' =>
+    'Категории',
+  '+ Attachment' =>
+    '+ Вложение',
+  '+ Link' =>
+    '+ Ссылка',
+  'Flags Management' =>
+    'Управление Флагами',
+  'Add Flag Level' =>
+    'Добавить Уровень Флага',
+  'New Base Level' =>
+    'Новый Уровень Базы',
+  'Keep Points' =>
+    'Очки Удержания',
+  'Capture points' =>
+    'Очки Захвата',
+  'All Base Levels' =>
+    'Все Уровни Базы',
+  'Base Level' =>
+    'Уровень Базы',
+  'Bases Management' =>
+    'Управление Базами',
+  'Add Base Level' =>
+    'Добавить Уровень Базы',
+  'New Category' =>
+    'Новая категория',
+  'Category: ' =>
+    'Категория: ',
+  'Categories Management' =>
+    'Управление Категориями',
+  'Add Category' =>
+    'Добавить Категорию',
+  'All Countries' =>
+    'Все Государства',
+  'In Use' =>
+    'Используется',
+  'In use' =>
+    'Используется',
+  'Not Used' =>
+    'Не Используется',
+  'Yes' =>
+    'Да',
+  'No' =>
+    'Нет',
+  'ISO Code' =>
+    'Код ISO',
+  'Countries Management' =>
+    'Управление Государствами',
+  'No Team Names' =>
+    'Нет Имен Команд',
+  'time' =>
+    'время',
+  'type' =>
+    'тип',
+  'pts' =>
+    'очки',
```

Comment:
Suggested Change: очк

Comment: Since there's a placeholder before this word, we should shortened it to be suitable for any number. Also for consistency